frontend/packages/pipelines-plugin/package.json

```diff
@@ -24,8 +24,7 @@
       "yamlTemplates": "src/templates/pipelines.ts",
       "catalog": "src/components/catalog",
       "topology": "src/topology",
-      "pipelineTabbedPage": "src/components/pipelines/PipelineTabbedPage.tsx",
-      "repositoryListComponent": "src/components/repository/list-page"
+      "pipelineComponents": "src/components"
```

Comment:
I think it's a good idea to separate exposed modules for separate areas to keep the chunk size smaller. I saw an earlier comment from @karthikjeeyar to merge it together into one module. This will increase the chunk size and load up everything at once. Maybe do not create so many modules. Try to group things together which you think should load together.